anchore_engine/db/entities/upgrade.py

```diff
@@ -1183,6 +1183,245 @@ def db_upgrade_012_013():
     upgrade_flush_centos_vulns_013()
 
 
+def remove_policy_engine_sizes():
+    """
+    Removes all varchar size hints for policy engine tables
+
+    :return:
+    """
+    engine = anchore_engine.db.entities.common.get_engine()
+
+    to_update = [
+        # Feeds
+        ('feeds', 'name'),
+        ('feeds', 'description'),
+
+        # Feed Groups
+        ('feed_groups', 'description'),
+        ('feed_groups', 'name'),
+
+        # GenericFeedDataRecord
+        ('feed_group_data', 'feed'),
+        ('feed_group_data', 'group'),
+        ('feed_group_data', 'id'),
+
+        # GemMetadata
+        ('feed_data_gem_packages', 'name'),
+        ('feed_data_gem_packages', 'latest'),
+
+        # NpmMetadata
+        ('feed_data_npm_packages', 'name'),
+        ('feed_data_npm_packages', 'sourcepkg'),
+        ('feed_data_npm_packages', 'latest'),
+
+        # Vulnerability
+        ('feed_data_vulnerabilities', 'id'),
+        ('feed_data_vulnerabilities', 'namespace_name'),
+        ('feed_data_vulnerabilities', 'link'),
+
+        # VulnerableArtifact
+        ('feed_data_vulnerabilities_vulnerable_artifacts', 'vulnerability_id'),
+        ('feed_data_vulnerabilities_vulnerable_artifacts', 'namespace_name'),
+        ('feed_data_vulnerabilities_vulnerable_artifacts', 'name'),
+        ('feed_data_vulnerabilities_vulnerable_artifacts', 'version'),
+        ('feed_data_vulnerabilities_vulnerable_artifacts', 'version_format'),
+        ('feed_data_vulnerabilities_vulnerable_artifacts', 'epochless_version'),
+
+        # FixedArtifact
+        ('feed_data_vulnerabilities_fixed_artifacts', 'vulnerability_id'),
+        ('feed_data_vulnerabilities_fixed_artifacts', 'namespace_name'),
+        ('feed_data_vulnerabilities_fixed_artifacts', 'name'),
+        ('feed_data_vulnerabilities_fixed_artifacts', 'version'),
+        ('feed_data_vulnerabilities_fixed_artifacts', 'version_format'),
+        ('feed_data_vulnerabilities_fixed_artifacts', 'epochless_version'),
+
+        # CpeVulnerability
+        # ('feed_data_cpe_vulnerabilities', 'feed_name'),
```

Comment:
should this be commented out?

Reply:
There was an issue in the postgres casting conversion on this table. This is a deprecated type (superceded by the nvdv2 tables) so I'm not worried about it not getting the update as this is also data that we control from the feed service rather than image content.